link/src/main/java/com/stripe/android/link/ui/LinkButtonView.kt

```diff
@@ -40,11 +40,16 @@ import com.stripe.android.link.LinkPaymentLauncher
 import com.stripe.android.link.R
 import com.stripe.android.link.theme.DefaultLinkTheme
 import com.stripe.android.link.theme.linkColors
+import com.stripe.android.uicore.StripeTheme
 
 private val LinkButtonVerticalPadding = 6.dp
 private val LinkButtonHorizontalPadding = 10.dp
-private val LinkButtonShape = RoundedCornerShape(22.dp)
-private val LinkButtonEmailShape = RoundedCornerShape(16.dp) // Button corner radius - padding
+private val LinkButtonShape = RoundedCornerShape(
+    StripeTheme.primaryButtonStyle.shape.cornerRadius.dp
+)
```

Comment:
This doesn’t seem to get updated correctly in the playground. Can you take a look at that?